rpcserver.go

```diff
@@ -1173,9 +1177,64 @@ out:
 	return nil
 }
 
-// fetchActiveChannel attempts to locate a channel identified by its channel
+// ForgetChannel removes all channel state from the database except for a
+// close summary. This method can be used to get rid of permanently unusable
+// channels dued to bugs fixed in newer versions of lnd. Only available
+// when lnd is started with the -debug flag.
+func (r *rpcServer) ForgetChannel(ctx context.Context,
+	in *lnrpc.ForgetChannelRequest) (*lnrpc.ForgetChannelResponse, error) {
+
+	if !cfg.Debug {
+		return nil, fmt.Errorf("ForgetChannel RPC call only " +
+			"available in debug mode")
+	}
+
+	index := in.ChannelPoint.OutputIndex
+	txidHash, err := getChanPointFundingTxid(in.GetChannelPoint())
+	if err != nil {
+		return nil, err
+	}
+	txid, err := chainhash.NewHash(txidHash)
+	if err != nil {
+		return nil, err
+	}
+	chanPoint := wire.NewOutPoint(txid, index)
+
+	dbChan, err := r.fetchOpenDbChannel(*chanPoint)
+	if err != nil {
+		return nil, err
+	}
+
+	summary := &channeldb.ChannelCloseSummary{
+		ChanPoint: *chanPoint,
+		ChainHash: dbChan.ChainHash,
+		RemotePub: dbChan.IdentityPub,
+		Capacity:  dbChan.Capacity,
+
+		// Appropriate type for forgotten channels?
+		CloseType: channeldb.LocalForceClose,
```

Comment:
Not sure what to do here

Reply:
Yeah a new close type would make sense ehre IMO.